lib/linux_admin/security/common.rb

```diff
@@ -0,0 +1,14 @@
+module LinuxAdmin
+  class Security
+    module Common
+      def replace_config_line(new_line, rep_regex, file_text)
+        new_text = file_text.gsub!(rep_regex, new_line)
```

Comment:
I don't think you want the ```!``` after the gsub since it will modify the original "file_text".